app/src/main/res/layout/fragment_my_data.xml

```diff
@@ -40,13 +40,14 @@
             app:layout_constraintTop_toBottomOf="@id/textDescription" />
 
         <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/data_list"
             android:layout_width="match_parent"
             android:layout_height="0dp"
             app:items="@{vm.items}"
             app:layoutId="@{R.layout.item_my_data}"
-            app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@id/header"
+            app:layout_constraintBottom_toTopOf="@id/enable_bluetooth_btn"
```

Comment:
Ja myslim, ze je cilem aby se to prekryvalo? Pripadne bych pod ten button pridal gradient z pruhledne bile do bile..

Reply:
Dá se to překrývání vyřešit jinak, než dát poslednímu prvku v tom RecyclerView paddingBottom?

Reply:
Rozumim..
Tohle vyresis tak, ze das tomu recyclerview padding bottom a nastavis clipToPadding na false.

Reply:
Je otazka jestli to delat vzhledem k tomuhle screenu https://trello-attachments.s3.amazonaws.com/5e6d1ff07c3ae91171206bc9/5e79abcf19fa34112edafce3/2e20d953859a3ce4d4e2070d3baa98a1/Screenshot_20200324-235445.png

Ale asi jo a idealne by se nejak zmensila cela ta sekce nahore

Reply:
clipToPadding jsem přesně potřeboval, díky. Už jsem na Androidu dlouho nedělal.

Reply:
Ta obrazovka už teď vypadá trochu jinak a není tam tolik údajů nahoře.

Reply:
Pravda